autogpt/config/config.py

```diff
@@ -219,6 +219,7 @@ def set_browse_chunk_max_length(self, value: int) -> None:
     def set_openai_api_key(self, value: str) -> None:
         """Set the OpenAI API key value."""
         self.openai_api_key = value
+        openai.api_key = value
```

Comment:
You do not need this assignment here.  All api calls are now made by explicitly passing in the api key from the configuration.

Reply:
yeah I saw that change thanks